compiler/spawn.nim

```diff
@@ -233,6 +238,10 @@ proc setupArgsForParallelism(g: ModuleGraph; n: PNode; objType: PType;
     let n = n[i]
     if i < formals.len and formals[i].typ.kind in {tyStatic, tyTypeDesc}:
       continue
+
+    if formals[i].typ.kind == tyVarargs and (formals[i].typ[0].kind in {tyTyped, tyUntyped} or 
+            formals[i].typ.kind in {tyTyped, tyUntyped}):
```

Comment:
Same. --> Factor out into a helper proc. Copy&paste is bad programming style.

Reply:
done